msalautomationapp/src/main/java/com/microsoft/identity/client/msal/automationapp/sdk/MsalSdk.java

```diff
@@ -175,6 +178,13 @@ public IAccount getAccount(@NonNull final Activity activity,
                 msalConfigResourceId
         );
 
+        //Handle accounts for AAD B2C
+        final Authority authority = pca.getConfiguration().getDefaultAuthority();
```

Comment:
What is the developer is not using the default authority? (devs can supply authority as part of AcquireTokenParameters...for our purposes, it is present on the `MsalAuthTestParams` )

Reply:
I made some changes addressing some points form the last conversation that we had 3b5863ab773d14ca4e6bbf8d25f1eb3df6decf41